File: Entities/QuestionTypes/NumberQuestion.cs
using System.ComponentModel.DataAnnotations;
using Core.Entities;

namespace Entities.QuestionTypes
{
    public class NumberQuestion : IEntity
    {
        [Key]
        public int QuestionId { get; set; }
        public bool IsRequired { get; set; }
        public bool IsMinNumber { get; set; }
        public bool IsMaxNumber { get; set; }
        public int MinNumber { get; set; }
        public int MaxNumber { get; set; }
    }
}

File: WebUI/UI/src/app/datas/models/question-types/number-question.ts
export class NumberQuestion {
  constructor(questionId: number, isRequired: boolean,  isMinNumber: boolean, isMaxNumber: boolean) {
    this.questionId = questionId;
    this.isRequired = isRequired;
    this.isMinNumber = isMinNumber;
    this.isMaxNumber = isMaxNumber;
  }

  questionId: number;
  isRequired: boolean;
  isMinNumber: boolean;
  isMaxNumber: boolean;
  minNumber: number;
  maxNumber: number;
}
